projects/largo/umbrella-setup.md

```diff
@@ -20,7 +20,78 @@ Most of this stuff works from your command line, which you get to through the de
 
 What's next?
 
-## 1. Clone the largo-umbrella git repository hosted at BitBucket.
+## Before you get started
+
+This guide assumes that you already have VVV setup on your machine. If you haven't setup VVV yet, follow the instructions on the [VVV Setup page](/staffing/onboarding/vvv-setup.md) before continuing.
+
+## 1. Download production database.
+
+Let's get the latest version of the WordPress database that powers the live, production sites.
+
+**NOTE**: Access to the INN/secrets repo is required.
+
+Do that with:
+```
+$ fab production wp.fetch_sql_dump
+```
+
+Go make another cup of tea, or two. At the time of this writing, the SQL dump file was 3.1 GB. This will take at least an hour.
+
+
+
+
+## 2. Create a new site using VV
+
+Run the following command from your vagrant root to initiate the vv site creation wizard:
+
+```
+vv create
+```
+
+Follow the prompts, entering the information below, or hitting [Enter] to skip.
+
+Prompt | Text to enter 
+------------ | -------------
+Name of new site directory: | largo-umbrella 
+Blueprint to use (leave blank for none or use largo): | *hit [Enter]* 
+Blueprint to use (leave blank for none or use largo): | *hit [Enter]*
+Domain to use (leave blank for largo-umbrella.dev): | *hit [Enter]*
+WordPress version to install (leave blank for latest version or trunk for trunk/nightly version): | *hit [Enter]*
+Install as multisite? (y/N): | y 
+Install as subdomain or subdirectory? : | subdomain 
+Git repo to clone as wp-content (leave blank to skip): | *hit [Enter]*
+Local SQL file to import for database (leave blank to skip): | *This directory must be an absolute path, so the easiest thing to do is to drag your mysql file into your terminal window here and the absolute filepath with fill itself in.*
+Remove default themes and plugins? (y/N): | N 
+Add sample content to site (y/N): | N 
+Enable WP_DEBUG and WP_DEBUG_LOG (y/N): | y
+ 
+This will take a few minutes to build your new site and load in the database, so be patient.
+
+## 3. Add the Largo Umbrella files to your new site
```

Comment:
Regarding this whole section: If one already has an existing umbrella repo under the old scheme, is it possible to link the old site's folder to the vvv htdocs folder using `ln -s` or similar, or is that approach inadvisable?